modules/repository-azure/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -9,6 +9,8 @@
 grant {
   // azure client opens socket connections for to access repository
   permission java.net.SocketPermission "*", "connect";
+  // TODO
```

Comment:
We shouldn't need this -- need to figure out how to make this work with the tmp directory federate token file in our test suite